tools/ci_connector_ops/ci_connector_ops/pipelines/commands/metadata_service.py

```diff
@@ -0,0 +1,73 @@
+import sys
+import click
+import anyio
+import dagger
+import logging
+
+from ci_connector_ops.pipelines.bases import Step, StepStatus, TestReport
+from ci_connector_ops.pipelines.actions import environments
+from ci_connector_ops.pipelines.contexts import PipelineContext
+from ci_connector_ops.pipelines.utils import (
+    DAGGER_CONFIG,
+    with_exit_code,
+)
+from rich.logging import RichHandler
+
+
+logging.basicConfig(level=logging.INFO, format="%(name)s: %(message)s", datefmt="[%X]", handlers=[RichHandler(rich_tracebacks=True)])
+logger = logging.getLogger(__name__)
+
+METADATA_LIB_MODULE_PATH = "airbyte-ci/connectors/metadata_service/lib"
+
+
+class MetadataLibRunTest(Step):
+    title = "Run Metadata Service Lib Unit Tests"
```

Comment:
nit for later: I'm wondering if we should add a `description` attribute that could explain to the world the philosophy of the implemented pipeline, with links to the thing under test code etc. It would be interesting metadata to propagate to logs or automatically generate docs.

Reply:
I like that idea! Feel free to add an issue for it!